docs/LibcSupport.md

```diff
@@ -15,8 +15,10 @@ limits.h | Yes | - |
 locale.h | Partial | Only basic support for C/POSIX locale |
 malloc.h | Partial | - |
 math.h | Partial | **Unsupported functions:** fmal(), tgamma() |
+netdb.h | Partial | Functions implicitly call out to untrusted host. **Supported functions:** getaddrinfo(), freeaddrinfo(), getnameinfo() |
 setjmp.h | Yes | - |
 signal.h | No | - |
+socket.h | Partial | Functions implicitly call out to untrusted host. <br> Full support available on Linux hosts. <br> **Unsupported functions on Windows hosts:**  recvmsg(), sendmsg(), socketpair() |
```

Comment:
What socket options & ioctls are supported/unsupported?  I'm skeptical that all are supported on windows.

Reply:
I am looking at https://pubs.opengroup.org/onlinepubs/007908799/xns/syssocket.h.html <sys/socket.h> and the Windows implementation in https://github.com/openenclave/openenclave/blob/master/host/windows/syscall.c. It appears to me that the socket calls in most cases can just be translated into equivalent Windows calls(no translation required)

Reply:
musl2bsd_socket_option in syscall.c shows only a very small number of socket options.  There are many socket options not listed.  (IP_ADD_SOURCE_MEMBERSHIP and many others that are part of POSIX and listed at https://www.man7.org/linux/man-pages/man7/ip.7.html and other pages - tcp, udp, etc.)

And you'll see that line 2477 of syscall.c points out that no ioctls are supported.

Reply:
Thanks! Will add more information.